packages/massa-web3/src/experimental/accountOperation.ts

```diff
@@ -0,0 +1,102 @@
+import { Account } from "./account"
+import { Address, Operation, OperationManager, OperationType, OptOpDetails, RollOperation, TransferOperation, calculateExpirePeriod } from "./basicElements"
+import { BlockchainClient } from "./client"
+
+
+/**
+ * A class regrouping all the account operations.
+ */
+export class AccountOperation {
+
+    constructor(public account: Account, public client: BlockchainClient) {}
+  
+    private async rollOperation( amount: number, opts?: OptOpDetails): Promise<string> {
+      const operation = new OperationManager(this.account.privateKey, this.client)
+      const details: RollOperation = {
+        // Todo: change with fetchMinimalFees once ready
+        fee: opts?.fee || 0,
```

Comment:
prefere the use of `??` operator here instead of `||`.  (even if the result will be the same in this case, its more coherent. )